akka-stream/src/main/protobuf/StreamRefContainers.proto

```diff
@@ -11,6 +11,11 @@ option optimize_for = SPEED;
 
 // TODO make explicit types for SinkRef/SourceRef as they can carry initial settings (initial demand etc)
 
+message SinkRef {
+    required ActorRef targetRef = 1;
+    optional int64 initialDemand = 2;
+}
```

Comment:
Thanks to this by sending over the sink ref, we already know how much the other side is safely ready to receive